Core/scripts/dirac-create-distribution-tarball.py

```diff
@@ -463,10 +458,9 @@ def __compileReleaseNotes( self, rstFile ):
     for baseFileName in baseList:
       htmlFileName = baseFileName + ".html"
       try:
-        fd = open( htmlFileName, "w" )
-        fd.write( parts[ 'whole' ] )
-        fd.close()
-      except Exception, excp:
+        with open( htmlFileName, "w" ) as fd:
+          fd.write( parts[ 'whole' ] )
+      except Exception as excp:
         return S_ERROR( "Could not write %s: %s" % ( htmlFileName, excp ) )
```

Comment:
In [7]: try:
   ...:     x = d[3] 
   ...: except Exception as x:
   ...:     pass
   ...: 

In [8]: x
Out[8]: KeyError(3)

In [9]: str(x)
Out[9]: '3'